components/compliance-service/reporting/relaxting/migrations.go

```diff
@@ -391,9 +391,9 @@ func (backend ES2Backend) reindex(src, dest, reindexScript, srcDocType string) (
 		Index(dest).
 		Type(mappings.DocType)
 	//source, err = reindexDestination.Source()
-	if err != nil {
-		return nil, indexToMigrateExists, errors.Wrap(err, fmt.Sprintf("%s unable to get reindexDest Source", myName))
-	}
+	//if err != nil {
+	//	return nil, indexToMigrateExists, errors.Wrap(err, fmt.Sprintf("%s unable to get reindexDest Source", myName))
+	//}
```

Comment:
Is there a reason we are not deleting this code?

Reply:
There is other commented out code there so I was following that convention, but I think I'll just delete it instead. 👍